test/v1alpha1/retry_test.go

```diff
@@ -43,6 +47,12 @@ func TestTaskRunRetry(t *testing.T) {
 	knativetest.CleanupOnInterrupt(func() { tearDown(ctx, t, c, namespace) }, t.Logf)
 	defer tearDown(ctx, t, c, namespace)
 
+	// Skip if the "embedded-status" feature flag has any value but "full".
```

Comment:
what's the reason for this?

Reply:
sounds good! did you have a chance to update this test?

Reply:
Yup, it's updated and now matches `test/retry_test.go`.